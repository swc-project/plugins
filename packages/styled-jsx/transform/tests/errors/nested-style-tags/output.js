import _JSXStyle from "styled-jsx/style";
const Component = ()=><>
    <header >
      <Comp >
        <_JSXStyle id={styles.__hash}>{styles}</_JSXStyle>
        <_JSXStyle id={"b3e1b6a7c9b96113"}>{""}</_JSXStyle>
      </Comp>
      <p className={"jsx-b3e1b6a7c9b96113"}><_JSXStyle id={"b3e1b6a7c9b96113"}>{""}</_JSXStyle></p>
    </header>
    <div className={"jsx-b3e1b6a7c9b96113"}>
      <h1 className={"jsx-b3e1b6a7c9b96113"}>
        Welcome!
        <style jsx>{``}</style>
      </h1>
      <_JSXStyle id={"b3e1b6a7c9b96113"}>{""}</_JSXStyle>
    </div>
    <div className={"jsx-b3e1b6a7c9b96113"}>
      <_JSXStyle id={"b3e1b6a7c9b96113"}>{""}</_JSXStyle>
      <h1 className={"jsx-b3e1b6a7c9b96113"}>
        Hello world!
        <style jsx global>{``}</style>
      </h1>
    </div>
    <div className={"jsx-b3e1b6a7c9b96113"}>
      <>
        <style jsx>{styles}</style>
      </>
      <_JSXStyle id={"b3e1b6a7c9b96113"}>{""}</_JSXStyle>
    </div>
  </>
;
