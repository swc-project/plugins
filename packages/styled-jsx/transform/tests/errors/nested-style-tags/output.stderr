
  x Detected nested styled-jsx tag.
  | Read more: https://nextjs.org/docs/messages/nested-styled-jsx-tags
    ,-[input.js:13:9]
 13 | <style jsx>{``}</style>
    : ^^^^^^^^^^^^^^^^^^^^^^^
    `----

  x Detected nested styled-jsx tag.
  | Read more: https://nextjs.org/docs/messages/nested-styled-jsx-tags
    ,-[input.js:21:9]
 21 | <style jsx global>{``}</style>
    : ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    `----

  x Detected nested styled-jsx tag.
  | Read more: https://nextjs.org/docs/messages/nested-styled-jsx-tags
    ,-[input.js:26:9]
 26 | <style jsx>{styles}</style>
    : ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    `----
